features/admin.core.v1/constants/i18n-constants.ts

```
@@ -400,6 +405,7 @@ export class I18nConstants {
         [ I18nConstants.FLOWS_NAMESPACE, "portals" ],
         [ I18nConstants.COMMON_USERS_NAMESPACE, "portals" ],
         [ I18nConstants.VERIFIABLE_CREDENTIALS_NAMESPACE, "portals" ]
+        [I18nConstants.CUSTOMER_DATA_SERVICE_NAMESPACE, "portals"]
```

ESLint

[error] Unexpected newline between object and [ of property access.

(no-unexpected-multiline)